shared/js/ui/models/whitelist.es6.js

```diff
@@ -46,9 +46,10 @@ Whitelist.prototype = window.$.extend({},
             // But first, strip the 'www.' part, otherwise getSubDomain will include it
             // and whitelisting won't work for that site
             url = url ? url.replace('www.', '') : ''
+            const parsedDomain = tldts.parse(url)
             const localDomain = url.match(/^localhost(:[0-9]+)?$/i) ? 'localhost' : null
```

Comment:
FWIW we are already not blocking for localhost, but we seem to still inject something, so probably good to keep for now.